tests/common.go

```diff
@@ -7792,6 +7792,39 @@ func GetPoolCapacityUsed(poolUUID string) (float64, error) {
 	return poolSizeUsed, nil
 }
 
+// GetRandomNode Gets Random node
+func GetRandomNode(pxNodes []node.Node) node.Node {
+	rand.Seed(time.Now().UnixNano())
+	randomIndex := rand.Intn(len(pxNodes))
+	randomNode := pxNodes[randomIndex]
+	return randomNode
+}
+
+func RemoveLabelsAllNodes(label string, forStorage, forStorageLess bool) {
+
+	if forStorage {
```

Comment:
you can traverse all the nodes at once rather than having separate loops for storage and storageless nodes

Reply:
Optimised this code to avoid an extra loop , we will now traverse the element  of PxNodes only added a check to get both all nodes i.e storage and storageless node via GetStorageDriverNodes()